src/components/Minimap/index.js

```diff
@@ -45,14 +46,22 @@ const Minimap = ({ data, isLoading, params, rowCount=4 }) => {
     const currentUrl = window.location.pathname
     const areaUrl = areaHits.length && areaHits[0].uri
     const isAreaActive = areaUrl === currentUrl
+    const hitPlural = t({
+      comment: "Pluralization of hit(s) for minimap boxes",
+      message: plural(areaHits.length, {one: "hit", other: "hits"})
+    })
+    const message = t({
+      comment: "Message displayed to jump to specific section within minimap",
+      message: `Jump to ${areaHits.length} ${hitPlural} in this area: ${hitTitles}`
+    })
     return (
       areaHits.length ?
       <a
-        key={idx}
-        href={appendParams(areaUrl, params)}
-        className={classnames('minimap__box', hitClass, { [`${hitClass}--active`] : isAreaActive})}
-        title={`Jump to ${areaHits.length} ${areaHits.length === 1 ? 'hit' : 'hits'} in this area: ${hitTitles}`} >
-        <span className='visually-hidden'>{`Jump to ${areaHits.length} ${areaHits.length === 1 ? 'hit' : 'hits'} in this area: ${hitTitles}`}</span>
+          key={idx}
+          href={appendParams(areaUrl, params)}
+          className={classnames('minimap__box', hitClass, { [`${hitClass}--active`]: isAreaActive })}
+          title={message} >
+        <span className='visually-hidden'>{message}</span>
```

Comment:
This is what it looks like within the code, when I define the translation outside of the return and use it within it.

Reply:
Yeah I think it's not always necessary, but in this case it really does clean things up a lot.